exampleCourse/elementExtensions/pl-file-editor/dot-preview/info.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "nodeModulesScripts": ["@viz-js/viz/lib/viz-standalone.js"],
```

Comment:
As a best practice, course elements shouldn't use `nodeModulesScripts`. The alternative is of course to check it in to the course repo. I don't love that, especially for the example course, but 🤷

Alternatively: build this functionality into the element instead of making it an extension.

I believe this was discussed elsewhere, but I don't love that this will load `viz-js` even when we will absolutely never use it on a page. This is a horrifically big file.